git-link.el

```diff
@@ -200,6 +203,7 @@ See its docs."
     ("gitorious" git-link-gitorious)
     ("gitlab" git-link-gitlab)
     ("git\\.\\(sv\\|savannah\\)\\.gnu\\.org" git-link-savannah)
+    ("opendev" git-link-opendev)
```

Comment:
Are we using a regular expression of `"opendev"` because it can be hosted on subdomains?

Reply:
no, the only domain I'm aware of is "opendev.org", should I use the full domain here?, I was just following what other providers where doing, so I'm all ears of what to do here :-)

Reply:
May has well be more specific. Can't recall if it's the case of all that do this but with GitHub and maybe GitLab because they have "enterprise" plans with subdomains.

Reply:
perfect, done.